### PR TITLE
Sqlite for development and test

### PR DESCRIPTION
**Pull request by coreyd303**

### CODE REVIEW

**Comment by kstack7** [comment]
Will need to compare this with #113 progress.

**Comment by kstack7** [comment]
@coreyd303 , just realized you're on #113 too...is this a new approach?

**Comment by coreyd303** [comment]
@kstack7 no, I had some mess that happened so I did a clean run. This is one that should be used

**Comment by kstack7** [comment]
Totally understood! I have a field day with git sometimes. Easy to keep working and overrun some commits :) Mind if I close #113 though?

**Comment by kstack7** [comment]
@coreyd303 ^^

File: .gitignore [hunk 1]
```diff
@@ -23,3 +23,7 @@
 
 # Ignore ReadTheDocs site style
 /docs/site
+
+# Ignore vendor / bundle
+/.bundle/
+/vendor/bundle
```
Comment (juliaelman) [inline .gitignore]: As mentioned in #139, please remove from this change from the PR. Should fix the merge conflict issues and be a cleaner merge.

File: config/database.yml [hunk 2]
```diff
@@ -56,7 +58,9 @@ development:
 # re-generated from your development database when you run "rake".
 # Do not set this db to the same as development or production.
 test:
-  <<: *default
+  adapter: sqlite3
```
Comment (juliaelman) [inline config/database.yml]: Need to clarify with @imuchnik, but we might need to leave this as default for tests to run correctly.

@coreyd303 did you run the test suite to verify that everything still runs smoothly?

**Comment by juliaelman** [comment]
@coreyd303 thanks for your PR! I've left a few notes on changes that need to be made to this PR prior to merging, plus one quick question. Please respond/verify at your earliest convenience!

**Comment by kstack7** [comment]
Hey @coreyd303! We're in the process of cleaning up some of our pre-launch PRs and wanted to see where you are with this. Please let us know if you are still working on this otherwise we will close it and may revisit after the launch. We really appreciate your help!!

**Comment by juliaelman** [comment]
Closing this issue for now, as we are not get a timely response from the original PR submitter.

@coreyd303 as @kstack7 mentioned, we're trying to clean our PR's right now. Feel free to re-submit your changes with our requests at any time! :fireworks:

**Pull request closed without merging**